app/models/datafile.rb

```diff
@@ -0,0 +1,13 @@
+class Datafile < ApplicationRecord
+  validates :origFilename,
+        :filename,
+        :realpath,
+        :directoryLabel,
+        :storageIdentifier,
+        :md5Hash,
```

Comment:
i think we're missing `mimeType` here (which is my fault, as it wasn't in the ER diagram but referenced in the spec)

Reply:
Good catch - yeah, I was focused on the diagram but remember now seeing mimetype in the meat and potatoes of the spec. I'll add that in.